test/jasmine/karma.conf.js

```diff
@@ -260,11 +260,52 @@ func.defaultConfig = {
         }
     },
 
-    browserify: {
-        transform: [pathToStrictD3, pathToShortcutPath],
-        extensions: ['.js'],
-        watch: !argv.nowatch,
-        debug: true
+    webpack: {
+        target: ['web', 'es5'],
```

Comment:
Have you considered targeting es6? https://betterprogramming.pub/do-we-still-need-to-compile-es6-javascript-code-into-es5-in-2022-f5ec70509e9b

Reply:
We still want to continuing support es5 syntax in our output.
But now the input syntax i.e. plotly.js source files as well as `node_modules` could be in es6 and further.

Reply:
@archmoj FWIW we just dropped IE support in Dash - and we did it in a minor https://github.com/plotly/dash/releases/tag/v2.7.0

I'd be comfortable doing the same in plotly.js, but open to discussing.

Reply:
I opened https://github.com/plotly/plotly.js/issues/6366. 
Let's investigate it & decide on that separately.